R/workspace.R

```diff
@@ -92,7 +94,8 @@ create_workspace <- function(
   app_insights = NULL,
   container_registry = NULL,
   exist_ok = FALSE,
-  show_output = TRUE) {
+  show_output = TRUE,
+  sku = 'basic') {
```

Comment:
[command]/bin/bash --noprofile --norc /__w/_temp/189c80f9-0899-4cbe-bac2-59bce2d14311.sh
[1] "./R/workspace.R"
/__w/1/s/R/workspace.R:98:9: style: Only use double-quotes.
  sku = 'basic') {
        ^~~~~~~
Error in check_code_style(directory = args[1]) : Code quality failed.
Execution halted